cnxepub/formatters.py

```diff
@@ -535,8 +460,7 @@ def _replace_exercises(elem, page_uuids):
 
 
 def render_exercise(exercise):
-    if len(exercise['items']) != 1:
-        raise Exception('Exercise "items" array is nonsingular')
+    assert len(exercise['items']) is 1, 'Exercise "items" array is nonsingular'
```

Comment:
[Using `is` to compare to literals "works" on CPython by accident](https://bugs.python.org/issue34850)